Makefile

```
@@ -35,9 +35,9 @@ generate: build # Generate CLI docs and samples
 	go run ./hack/generate/samples/generate_testdata.go
 
 .PHONY: bindata
-OLM_VERSION=0.15.1
+OLM_VERSIONS="0.16.1 0.15.1 0.14.1 0.13.0 0.12.0"
```

Perhaps we could just add versions starting from 0.15.1? Also don't use quotes in Makefile variables if you can avoid them (they're interpreted literally).
```suggestion
OLM_VERSIONS = 0.16.1 0.15.1
```